packages/react-composites/src/composites/CallComposite/pages/LobbyPage.tsx

```diff
@@ -76,3 +109,17 @@ const overlayPropsWaitingToBeAdmitted = (strings: CallCompositeStrings): LobbyOv
   moreDetails: strings.lobbyScreenWaitingToBeAdmittedMoreDetails,
   overlayIcon: <CallCompositeIcon iconName="LobbyScreenWaitingToBeAdmitted" />
 });
+
+/* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
+const overlayPropsOutboundCall = (
+  strings: CallCompositeStrings,
+  participant: RemoteParticipantState
+): LobbyOverlayProps => {
+  const title = isPhoneNumberIdentifier(participant.identifier)
```

Comment:
nit: The logic will be more readable if you separate out the two cases entirely:

```
if (isPhoneNumerIdentifier(participant.identifier)) {
  return {
    title: ...
    moreDetails: ...
  }
}

return {
  title: ...
  moreDetails: ...
}
```